README.md

```diff
@@ -136,6 +136,51 @@ This endpoint returns a list of all existing categories. It will return an empty
 This endpoint returns one category corresponding to the id provided in the request. If no category with that
 id exists, it will return a 404 response code.
 
+### Subcategories
```

Comment:
@egiurleo I'm not sure if you're working on a new format for the API docs - feel free to rework this!